graphics/macgui/macwindowborder.h

```diff
@@ -29,9 +29,23 @@
 #include "graphics/nine_patch.h"
 #include "graphics/managed_surface.h"
 #include "graphics/transparent_surface.h"
+#include "graphics/macgui/macfontmanager.h"
```

Comment:
In general, avoid using includes in other includes unless absolutely necessary.

I see that these three added includes could go to macwindowborder.cpp

Also, please remove nine_patch.h from here and use forward declaration instead, e.g. add: `class NinePatchBitmap;` after you forward-declare `MacWindow`, because you are using only pointers here.

This way, compilation goes faster and less files get recompiled when you change a header.